src/config/kernel.md

```diff
@@ -76,15 +88,27 @@ omit_drivers+=" radeon "
 ```
 
 Now initramfs needs to be regenerated to make the changes take effect on the
-next reboot.
+next reboot:
 
 ```
 # dracut --force
 ```
 
 #### mkinitcpio
 
-XXX: add example of blacklisting kernel modules for mkinitcpio
+To blacklist modules from being included in a mkinitcpio initramfs 
```

Comment:
This is all duplicated from a previous section. You could just reference it, like: 

'After configuring modules to be blacklisted, some may still be loaded by the `initramfs`, in which case a new `initramfs` needs to be generated for the changes to take effect:'

Reply:
Or I guess modules could be *added* also, but something like that.

Reply:
In my opinion the duplication justified, because those using mkinitcpio would just skip over the dracut part.